objectivec/Tests/GPBMessageTests+Serialization.m

```diff
@@ -955,6 +966,16 @@ - (void)testErrorRecursionLimitReached {
   XCTAssertEqual(error.code, GPBCodedInputStreamErrorRecursionDepthExceeded);
 }
 
+- (void)testErrorMissingRequiredField {
```

Comment:
This one needs to be wrapped in a #if DEBUG gate, we currently only check for required fields in DEBUG build to avoid the slowdown in production apps.  (or we should always test for it, but that means generating an NSData also needs to have an error param, so likely a larger change that this cl.)